### PR TITLE
Release 6.0.0 versioning

### PR DESCRIPTION
**Pull request by maddieclayton**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
sign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/472/

**Comment by maddieclayton** [comment]
updated sign job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/473/

File: src/ResourceManager/Storage/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+
+## Version 5.0.0
```
[the remark this answers is not in the document]
Comment (maddieclayton) [reply src/ResourceManager/Storage/ChangeLog.md]: There are breaking changes in the current breaking changes file for May.

File: src/Storage/Azure.Storage.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '4.2.1'
+ModuleVersion = '4.2.2'
```
[the remark this answers is not in the document]
Comment (maddieclayton) [reply src/Storage/Azure.Storage.psd1]: done

**Comment by maddieclayton** [comment]
sign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/481/

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/485/

File: ChangeLog.md [hunk 3]
```diff
@@ -1,4 +1,107 @@
-﻿## 5.7.0 - April 2018
+﻿## 6.0.0 - May 2018
+#### General
+* Set minimum dependency of modules to PowerShell 5.0
+
+#### Azure.Storage
+* Support  as Storage blob container name
```
[the remark this answers is not in the document]
Comment (maddieclayton) [reply ChangeLog.md]: They were put in the changelog and weren't included in last release, so I think they did happen since last release.

**Comment by maddieclayton** [comment]
@markcowl Updated per currently merged PRs. Could use a quick review.

**Pull request closed without merging**